sdk/communication/communication-chat/src/signaling/signalingClient.browser.ts

```diff
@@ -1,13 +1,34 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { CommunicationSignalingClient, SignalingClient } from "@azure/communication-signaling";
+import {
+  CommunicationSignalingClient,
+  CommunicationUserCredential,
+  SignalingClient
+} from "@azure/communication-signaling";
 import { CommunicationTokenCredential } from "@azure/communication-common";
 import { AzureLogger } from "@azure/logger";
+import { AbortSignalLike, AccessToken } from "@azure/core-http";
 
 export const getSignalingClient = (
   credential: CommunicationTokenCredential,
   logger: AzureLogger
 ): SignalingClient | undefined => {
-  return new CommunicationSignalingClient(credential, logger);
+  return new CommunicationSignalingClient(new SignalingCredentialType(credential), logger);
 };
+
+/**
+ * Bridge credential until @azure/communication-signaling has been updated
+ * to match common's CommunicationTokenCredential interface.
+ */
+class SignalingCredentialType implements CommunicationUserCredential {
```

Comment:
@LuChen-Microsoft Had to do this temporarily, we need to fix this inside the signaling package